content/r-package-dev/Documentation.Rmd

```diff
@@ -262,11 +304,17 @@ etc ...
 
 ```
 
-Save your new R Markdown file. The next time you build your package, your vignette should now be available!
+Save your new R Markdown file. To get the vignette to show up in your R package, you then need to:
+
+```{r eval=FALSE}
+devtools::build_vignettes()
+```
+
+This will create the html output, an R script that is just the executable R code in the vignette, and a copy of the markdown file, moving all 3 of those files into the inst/doc folder.
```

Comment:
I'd like to add a note about the build options here. The reason I said "the next time you build" was because I showed how to add vignettes to the build options. Would it be worth saying, "if you checked "Vignettes" for Roxygen options in the build tool configurations earlier [LINK], then you can build your vignette at the same time as your packages." or something?